go.mod

```diff
@@ -242,17 +242,10 @@ require (
 
 replace (
 	// pinned because of incompatibility with k8s.io/apiserver
-	github.com/google/cel-go => github.com/google/cel-go v0.22.1
 	github.com/openshift/client-go => github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0 // release-4.5
 
 	// use staged repositories
 	github.com/operator-framework/api => ./staging/api
 	github.com/operator-framework/operator-lifecycle-manager => ./staging/operator-lifecycle-manager
 	github.com/operator-framework/operator-registry => ./staging/operator-registry
-
-	// pinned because of incompatibility with k8s.io/apiserver otelgrpc API changes
-	go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc => go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc v0.58.0
```

Comment:
We remove the replace of grcp
We are using the latest 
We have no longer this replace in upstream
See that we are using latest: https://github.com/operator-framework/operator-lifecycle-manager/blob/961df6c58777205d5fb872a05eb27bf45269dc25/go.mod#L147